### DISCUSSION TITLE
smartenapplianceenabler.service failed - bei manueller Installation

### DISCUSSION THREAD

**Discussion by lissi514** [camueller/SmartApplianceEnabler#340] [Installation und Betrieb]
Hallo Forum,

ich habe die manuelle Installation vorgenommen. 

Bei der Status-Prüfung bekomme ich die Meldung, dass der Service/Dienst nicht gestartet ist: 

"Job for smartapplianceenabler.service failed becaus teh control prosess exited with error code. See "systemctl status smartapplianceenabler.service" and "journalctl -xe" for details"

An dieser Stelle komme ich nicht weiter, weil ich die Fundstelle nicht finde und damit keinen Zugang zur Lösung erhalte.

Kann mir jemand auf die Sprünge helfen? Vielen Dank für Eure Unterstützung.

VG Peter

**Comment by camueller** [comment]
Die manuelle Installation ist nur für diejenigen gedacht, die sich mit Linux auskennen, was hier offensichtlich nicht der Fall ist.
Daher sollte die [Standard-Installation](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md) oder die seit kurzem dokumentierte [Installation via Script](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/InstallationViaScript_DE.md) gewählt werden.

**Reply by lissi514** [reply]
Guten Morgen, da haben Sie Recht. Vielleicht reicht meine Kompetenz für die Standard-Installation auch nicht. Denn hier bin ich auch trotz mehrfacher Versuche (auf einem Raspbi 3 B+) erfolgreich gescheitert...

**Reply by camueller** [reply]
Fehler bei der Standard-Installation waren bisher immer darauf zurückzuführen, dass die Anleitung nicht exakt umgesetzt wurde (vor allem Windows-User tun sich da schwer). In diesen Fällen kann die Installation via Script eine Vereinfachung sein.

**Comment by lissi514** [comment]
Ja, es ist nicht so, dass ich mich nicht mit Ihren Ausführungen auseinandergesetzt hätte. Ich bleibe dran und berichte...

**Comment by lissi514** [comment]
So, funzt! Und Entschuldigung!

Jetzt kümmer ich mich weiter...

![PXL_20220915_110945297](https://user-images.githubusercontent.com/113547561/190389312-d84bd84b-2fac-4e76-a8b8-db582ac1ccc4.jpg)
